draft-ietf-rats-eat.md

```diff
@@ -417,23 +417,26 @@ Endorsement:
 
 # Top-Level Token Definition
 
-An EAT is a "message", a "token", or such whose content is a Claims-Set about an entity or some number of entities.
-An EAT MUST always contains a Claims-Set.
+An EAT MUST contain a claims set about an entity packaged as a CWT {{RFC8392}} or JWT {{RFC7519}}. This specification extends {{RFC8392}} and {{RFC7519}} by defining additional claims for attestation. All definitions, requirements, creation and validation procedures, security considerations, IANA registrations, etc. from CWT and JWT apply to the EAT format.
 
-An EAT may be encoded in CBOR or JSON as defined here.
-While not encouraged, other documents may define EAT encoding in other formats.
+Authenticity and integrity protection MUST be provided for EATs. This document defines the use of CWT or JWT for this purpose.
```

Comment:
Prefer "relies on" rather than "defines the use of".

Reply:
If you consider UCCS, then these two sentences are not correct since with UCCS integrity and authenticity is not provided by the CWT/JWT

Reply:
The next sentence is "Extensions to this specification MAY use other methods of protection."

Reply:
Seems OK to me now.